enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,205 @@
+
+
+### API Extensions
+- No API changes are required. 
+- See https://github.com/openshift/enhancements/blob/master/enhancements/etcd/automated-backups.md#api-extensions
+
+### Topology Considerations
+TBD
+#### Hypershift / Hosted Control Planes
+TBD
+#### Standalone Clusters
+TBD
+#### Single-node Deployments or MicroShift
+TBD
+
+### Implementation Details/Notes/Constraints [optional]
+- Need to agree on a default schedule.
+- Need to agree on a default retention policy.
+
+- Several options exist for the default `PVCName`.
+  - Relying on `dynamic provisioning` is sufficient, however not an option for `SNO` or `BM` clusters.
+  - Utilising `local storage operator` is a proper solution, however installing a whole operator is too much overhead.
+  - The most viable solution to cover all OCP variants is to use `local volume`.
+    Please find below this solution's Pros & Cons.
+    - Pros :- 
+      - The PV will be bound to a specific master node.
+      - The Backup Pod will be scheduled always to this master node since it uses a PVC bound to this PV.
+      - Using an SSD is possible, since local volume allows mounting it into a Pod.
+      - Having the Backup Pod scheduling deterministic is vital, since the retention policy needs the recent backup in place to work.
+    - Cons :-
+      - If the master node where the PV is mounted became unhealthy/unavailable/unreachable etc.
+      - The backups are no longer accessible, also taking new backups is no longer possible.
+      - Ideally, the backups should be taken on a round-robin fashion on each master node to avoid overwhelming a specific etcd member more than others.
+      - Also spreading the backups among all etcd cluster members provide guarantees for disaster recovery in case of losing two members at the same time.
+      - However, using the `local volume` option over all the master nodes will be complicated and error prune.
+
+### Risks and Mitigations
+
+When the backups are configured to be saved to a `local` type PV, the backups are all saved to a single master node where the PV is provisioned on the local disk.
+
+In the event of the node becoming inaccessible or unscheduled, the recurring backups would not be scheduled. The periodic backup config would have to be recreated or updated with a different PVC that allows for a new PV to be provisioned on a node that is healthy.
+
+If we were to use `localVolume`, we need to find a solution for balancing the backups across the master nodes, some ideas could be
+
+- create a PV for each master node using `localVolume` and the backup controller from CEO should take care of balancing the backups across the `available` and `healthy` volumes.
+- the controller should keep the most recent backup on a healthy node available for restoration.
+- the controller should skip an unhealthy master node from taking a backup
+
+
+### Drawbacks
+
+## Design Details
+
+- Add default values to current `Etcdbackup Spec` using annotation.
```

Comment:
@openshift/openshift-team-etcd I added this section after discussion on the `Architecture Call`

Input is from @soltysh @jsafrane 

- It is recommended to use `default` values on current API.
- Do not rely on CVO on managing the default config.
- Use storage solution based on OCP  :- 
  - Dynamic provisioning on Cloud based.
  - Local Volume for SNO / BM.

Reply:
let's say we have to restore from a snapshot, how can we access the local volume?

Reply:
The general flow should be like follows:
1. All the EtcdBackup fields are defaulted, where possible. That's why I'm proposing to put this functionality behind already existing `AutomatedEtcdBackup` feature gate, rather than introducing a new one. 
2. The `pvcName` will be the one that will have to by populated automatically by the CEO, unless a users sets one. It will use:
  - the available default storage class in the cluster (storage team can provide you with where you should be looking at it);
  - if above is not available, fallback to using a localVolume, as explained by Hemant above;
  - add a warning condition to the CEO (at least for that localVolume), reporting that we're working off of default etcd backup, and that the cluster admin should verify correctness of that configuration.